packages/appstore/src/modules/wallets/wallets.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect } from 'react';
+import { Loading, ThemedScrollbars } from '@deriv/components';
+import { useActiveWallet, useWalletsList } from '@deriv/hooks';
+import { observer, useStore } from '@deriv/stores';
+import AddMoreWallets from 'Components/add-more-wallets';
+import ModalManager from 'Components/modals/modal-manager';
+import DesktopWalletsList from './desktop-wallets-list';
+import MobileWalletsCarousel from './mobile-wallets-carousel';
+import './wallets.scss';
+
+const Wallets = observer(() => {
+    const { client, ui } = useStore();
+    const { switchAccount, is_authorize } = client;
+    const { is_mobile } = ui;
+    const { data } = useWalletsList();
+    const active_wallet = useActiveWallet();
+
+    useEffect(() => {
+        if (!active_wallet && data && data?.length) {
+            switchAccount(data[0].loginid);
```

Comment:
@farzin-deriv we need to switch to the corresponding wallet of the trading account, not the first wallet. But this is out of scope here, you can ignore it for now.

Reply:
@mahdiyeh-deriv Ah you mean using that `linked_to` thing right? 🤔 I will do this in the next PR 🙇🏻